luaui/configs/bar_hotkeys_60.lua

```diff
@@ -1,358 +1,235 @@
--- BAR hotkey config file: default for 60% keyboards
--- remap F-keys to use meta+ (spacebar) and ` to Q with modifiers
-local bindings = {
-	{            "esc", "quitmessage"                },
-	{      "Shift+esc", "quitmenu"                   },
-	{ "Ctrl+Shift+esc", "quitforce"                  },
-	{  "Alt+Shift+esc", "reloadforce"                },
-	{     "Any+escape", "edit_escape"                },
-	{      "Any+pause", "pause"                      },
-	{            "esc", "teamstatus_close"           },
-	{            "esc", "customgameinfo_close"       },
-	{            "esc", "buildmenu_pregame_deselect" },
+-- BAR hotkey config file: optimized for use with grid build menu and 60% keyboards.
+-- remaps F-keys to use meta+ (spacebar) and moves commands from ` to sc_q with modifiers.
+	local bindings = {
```

Comment:
Ah, just indentation, if you bring indentation back I'm sure it will fix the diff. On notepad++ you can select the whole block and `shift+tab` or something similar.